src/api/stats/telos/index.js

```diff
@@ -1,10 +1,7 @@
 const getOmnidexLpApys = require('./getOmnidexLpApys');
 const getZappyLpApys = require('./getZappyLpApys');
 
-const getApys = [
-  getOmnidexLpApys, 
-  getZappyLpApys
-];
+const getApys = [getOmnidexLpApys, getZappyLpApys];
```

Comment:
Nit: can we change this back? More clear that way, think this is done by the prettierrc